public/stylesheets/app.css

```diff
@@ -153,10 +154,12 @@ body.dark-mode main p,
 body.dark-mode main label,
 body.dark-mode main strong,
 body.dark-mode main a {
-	color: var(--dark-mode-fg-color);
+  color: var(--dark-mode-fg-color);
 }
 
-.label.is-required::after {
-	content: ' *';
-	color: red;
-  }
\ No newline at end of file
+.control.is-required::after{
+  content: "Required";
```

Comment:
Shouldn't this be an asterisk following the label? The `Required` after the field looks a little odd.
![localhost_1337_nmap (1)](https://github.com/ronin-rb/ronin-app/assets/12671/eb42dd1f-9313-4ac0-ac66-db47133f6649)